fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -1106,18 +1106,20 @@ private void checkAndAddColumn(List<Column> modIndexSchema, Column newColumn, Co
             hasPos = true;
         }
 
-        newColumn.setUniqueId(newColumnUniqueId);
+        // newColumn may add to baseIndex or rollups, so we need copy before change UniqueId
```

Comment:
Does doris support add a column to rollup and the column is not in base table?

Reply:
not support.

now rollup columns are subset of base table.